src/OidcClient/OidcClient.csproj

```diff
@@ -4,14 +4,14 @@
     <PackageId>IdentityModel.OidcClient</PackageId>
     <RootNamespace>IdentityModel.OidcClient</RootNamespace>
     <AssemblyName>IdentityModel.OidcClient</AssemblyName>
-    
-    <TargetFramework>netstandard2.0</TargetFramework>
-    
+
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
I'm pretty hesitant to change TargetFrameworks. I assume some of the json serialization options are missing in netstandard2.0? What are we missing?